doc/source/data/index.rst

```diff
@@ -45,3 +45,17 @@ util module:
    :maxdepth: 1
 
    util
+
+SunPy and AstroPy Integration
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+HelioPy is built to be used alongside SunPy and AstroPy. Before V0.6, HelioPy returned
```

Comment:
- Can SunPy and AstroPy get links to their home pages
- V0.6 -> v0.6 (I think it looks better small case)